packages/website/docs/creating-a-renderer.md

```diff
@@ -1,9 +1,15 @@
 ---
 id: creating-a-renderer
-title: Creating a Renderer
+title: Creating a custom Renderer
 ---
 
-Learn how to build an autocomplete UI using React.
+Learn how to build a full autocomplete UI from the ground up.
+
+:::info You might not need a custom renderer
+
+Building a custom renderer is an advanced pattern that leverages the [`autocomplete-core`](createAutocomplete) package to fully control the rendered experience. You shouldn't need to use it unless you've reached limitations with [`autocomplete-js`](autocomplete-js) and it's templating capabilities.
+
+:::
```

Comment:
Just saw it on the preview, maybe this isn't the best idea to make this the first thing on the page. I'd probably move it right before the first heading, and after the intro. People will still see it early, but it's not as brutal for those who know they need it.

Reply:
See 5168534dd013e1f0462366c9c7a7d4869ceeb0f6.